rllib/utils/test_utils.py

```diff
@@ -1401,7 +1401,7 @@ def check_supported_spaces(
     config: "AlgorithmConfig",
     train: bool = True,
     check_bounds: bool = False,
-    frameworks: set = None,
+    frameworks: Optional[Sequence[str]] = None,
```

Comment:
Can we please stop using `Sequence` (or `Mapping` for that matter)? A `Sequence` could also be a str. Instead, let's be more clear as to what we expect from the user. List is fine here.

Unrelated to this line: `Mapping` is also not great (I see this all over the place these days in RLlib). A Mapping per-se doesn't have the `update()` API for example, so IDEs consider these warnings and mark the code yellow if a method uses `.update` (or `get` or any other dict-specific API) on a var that was declared Mapping.

Reply:
Fair. Sequence is what was used for framework_iterator(). I'll change that here.